src/index.ts

```diff
@@ -26,6 +26,7 @@ export interface WatchConfiguration {
 export interface Configuration extends Omit<BuildOptions, 'watch'> {
   packager: 'npm' | 'yarn';
   packagePath: string;
+  rootPackagePath: string;
```

Comment:
Can do! I added it because `packagePath` is now only for the local package.json, so if someone needed to change where the root/monorepo package.json is, they can do so with this new option. To be honest, I'm not sure if anybody would actually need this. It's utilized here:

https://github.com/floydspace/serverless-esbuild/pull/94/files#diff-f0bb16aaf30a795796352d0a18a573bb97b4890fa618deb64662dab5007fcb53R189

I'll add a short description to the Configure section, but if you think it's unnecessary I can take it out.

Reply:
I remember that I added a `packagePath` in https://github.com/floydspace/serverless-esbuild/issues/80? it's why I was a bit confused. Maybe naming is not perfect and should be related more to npm/yarn workspaces.
 I'm okay if you leave it, but would be nice have it documented. thank you.

Reply:
Ah right. I think these changes would make `packagePath` redundant, then. The difference between that issue and mine was they didn't have nested/local package.jsons, whereas I did. 

I'll take out this option, as it doesn't make a whole lot of sense with what `packagePath` was intended for. But I will add some documentation for packager/packagePath configs since they are undocumented.

Reply:
thank you, go ahead

Reply:
should be good to go!